awesomemotive/affiliatewp-affiliate-product-rates
# Issue/48 - Bump WordPress minimum requirements

Issue #48


**Housekeeping**


**Proposed changes**


## How to test it
1. Install and activate Affiliate Product Rates add-on
2. Confirm the plugin works as expected without any PHP errors.
3. Confirm existence and test saving the new settings of Product Rates when editing an affiliate
